browsepy/compat.py

```diff
@@ -248,7 +248,14 @@ def pathconf(path,
     '''
 
     if pathconf_fnc and pathconf_names:
-        return {key: pathconf_fnc(path, key) for key in pathconf_names}
+        pathconf_output = {}
+        for key in pathconf_names:
+            try:
+                pathconf_output[key] = pathconf_fnc(path, key)
+            except OSError:
+                print(f"pathconf key {key} unsupported for this platform")
+                pass
```

Comment:
Please check `OSError.errno` attribute equals `errno.EINVAL` to ensure only that error code is ignored, reraising everything else.